<launch>

    <node pkg="rss_control" type="a_star_node" name="a_star" output="screen">
        <param name="debug" value="true"/>
        <param name="odom_in_topic" value="/odom"/>
        <param name="map_in_topic" value="/map"/>
        <param name="target_point_topic" value="/target"/>
        <param name="robot_radius" value="0.25"/>
    </node>

</launch>